### PR TITLE
717 precourse beta

### LINKED ISSUE #717: PreCourse Beta Landing page
Simple landing page for communicating the precourse beta information

### PR DESCRIPTION
**Pull request by nikeshashar**
V1 of the PreCourse Beta landing page. 
Closes #717

### CODE REVIEW

**Comment by nikeshashar** [comment]
👍 from me

**Comment by nikeshashar** [comment]
:shipit:

File: source/precourse-beta-application.html.haml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- @hide_nav = true
+- @hide_from_google = true
+
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/student-concentrating.jpg")});"}
+  .container
+    .centered-row
+      %h1 PreCourse Beta
+      %p Please complete the form below
+%section.typeform
+  .typeform-widget{"data-url" => "https://makersacademy.typeform.com/to/rJiR8b"}
```
Comment (jafrog) [inline source/precourse-beta-application.html.haml]: Should it still be here?

Comment (anitacanita) [inline source/precourse-beta-application.html.haml]: No, I don't think so. Should have deleted it.

**Comment by jafrog** [comment]
:shipit:

**Pull request merged**